### ISSUE TITLE
[Adaptive Cards- Web Render Test on Teams]: Voice Over is not announcing the changed state as 'Selected' when user is selecting the radio button using enter or spacebar key.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#412]
### Target Platforms

iOS

### SDK Version

7.13

### Application Name

Microsoft Teams

### Problem Description

**Test Environment:**
Device: iPhone 12
iOS Version: 18.5
Teams Application Version: 7.13
Screen Reader: Voice Over
Keyboard: Logitech K480

**Pre-Requisite:**
Navigate through keyboard+voice over only.

**Repro Steps:**
1. Open the application using double tap.
2. Press tab key and navigate to the AC Web render group, and press enter key to enter in the group.
3. Press tab key and navigate the radio button present on the polls.
4. Press spacebar key and observe the issue.

**Actual Result:**
Voice over is remain silent when user is selecting any radio button option using enter key. Here, when user is selecting any radio button then screen reader remain silent and it does not announce the changed state information as 'Selected'.

**Observations:**
This issue is working fine while navigating using swipe gesture only.

**Expected Result:**
Voice over should announce the changed state information as 'Selected' while selecting any radio button using enter or spacebar key.

**User Impact:**
It will impact the keyboard user as they will get confused if the keyboard focus lands on the non-intearctive control as they will assumes as this is interactve control and they will try to perform action on it.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/a3465b8e-1609-4850-a9b4-b45eeb49e108

### Card JSON

```JSON
NA
```
